packages/cli/index.js

```diff
@@ -3,7 +3,12 @@
 import * as fs from 'fs/promises';
 import * as path from 'path';
 import * as url from 'url';
-import { $, ProcessOutput } from 'zx';
+import download from 'download';
```

Comment:
Mostly a matter of personal taste, but I try to keep node's internal libs listed first, so I suggest something like:
```js
import { exec, spawn } from 'child_process';
import * as fs from 'fs/promises';
import * as path from 'path';
import * as url from 'url';
import { promisify } from 'util';
import download from 'download';
import which from 'which';

const execAsync = promisify(exec);
const spawnAsync = promisify(spawn);
```